### PR TITLE
add br.com.gualandi.Xjump

### PR DESCRIPTION
**Pull request by hugomg**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: br.com.gualandi.Xjump.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+modules:
+  - name: xjump
+    buildsystem: simple
+    build-commands:
+      - ./configure --prefix=/app
+      - make
+      - make install
```
Comment (hfiguiere) [inline br.com.gualandi.Xjump.yml:16]:
```suggestion
```

autotools is the default and should just work.

File: br.com.gualandi.Xjump.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+app-id: br.com.gualandi.Xjump
```
Comment (hfiguiere) [inline br.com.gualandi.Xjump.yml:1]: Is this domain owned by the developer? it seems to redirect to something else.

**Review by hfiguiere — CHANGES_REQUESTED** [review]
There don't seem to be neither an icon, a desktop file nor an appstream file, so the build will definitely fail.

File: br.com.gualandi.Xjump.yml (hunk 1 above)
Comment (hugomg) [reply br.com.gualandi.Xjump.yml:16]: The project technically doesn't use autotools, it is a custom configure script. Can I still use "buildsystem:autotools" in that case?

File: br.com.gualandi.Xjump.yml (hunk 2 above)
Comment (hugomg) [reply br.com.gualandi.Xjump.yml:1]: I do own the domain. I concede that the redirect is confusing though.

**Comment by hugomg** [comment]
Hi, thanks for the prompt review. The icon, desktop file, and appstream xml are inside the "misc" directory in the upstream repository. Do I need to copy them to this PR? The build worked when I tested on my machine, at least. https://github.com/hugomg/xjump-sdl/tree/main/misc

**Comment by hfiguiere** [comment]
bot, build br.com.gualandi.Xjump

**Comment by hfiguiere** [comment]
> Hi, thanks for the prompt review. The icon, desktop file, and appstream xml are inside the "misc" directory in the upstream repository. Do I need to copy them to this PR? The build worked when I tested on my machine, at least. https://github.com/hugomg/xjump-sdl/tree/main/misc

sorry I missed them. It's all good.

File: br.com.gualandi.Xjump.yml (hunk 1 above)
Comment (nedrichards) [reply br.com.gualandi.Xjump.yml:16]: should be fine and is very easy to test ;-)

Comment (hugomg) [reply br.com.gualandi.Xjump.yml]: Indeed, I tested it here and the 'autotools' setting invoked passed exactly the same flags to ./configure. I have pushed a commit to switch to that. Thanks for the tip!

**Comment by barthalion** [comment]
bot, build br.com.gualandi.Xjump

**Comment by hugomg** [comment]
Thanks! 👏

**Pull request closed without merging**